web/static/js/app.js

```diff
@@ -6,13 +6,13 @@ import { render } from "react-dom"
 import RemoteRetro from "./components/remote_retro"
 import RetroChannel from "./services/retro_channel"
 
-const retroChannelConfiguration = {
+const remoteRetroProps = {
```

Comment:
not sure i like this rename. this object is really the retroChannelConfiguration, and at this point, there's no need to pass `retroUUID` as a prop to the `RemoteRetro` component, as it goes unused. Can we revert this in favor of explicitly passing what's needed?